nototools/noto_fonts.py

```diff
@@ -419,6 +419,9 @@ def wws_family_id_to_name_parts(wws_id):
       parts.append('kr')
   elif script == 'sym2':
     parts.append('Symbols2')
+  elif script == 'phag':
+    # allow hyphenated name in name table
+    pargs.append('Phags-pa')
```

Comment:
pargs?  Isn't it |parts|?

Reply:
Um... this must not have executed in the tools I ran.  Hmmm.  I'll leave this in and fix it instead of removing it entirely.